pkg/controllers/rollout/controller_integration_test.go

```diff
@@ -177,6 +177,10 @@ var _ = Describe("Test the rollout Controller", func() {
 			By(fmt.Sprintf("resource binding `%s` is marked as not scheduled", binding.Name))
 			deletedBindings = append(deletedBindings, binding)
 		}
+		// simulate that some of the bindings are applied
```

Comment:
should we move the for loop before we set them as unscheduled?
Could you please add a comment to indicate why we move these statement?

otherwise, the rollout controller will try to delete the bindings before we mark them as applied.